ament_uncrustify/ament_uncrustify/main.py

```diff
@@ -282,8 +203,12 @@ def find_executable(file_name, additional_paths=None):
     return shutil.which(file_name, path=path)
 
 
-def get_files(paths, extensions, excludes=[]):
-    files = []
+def get_files(paths, extension_types, excludes=[]):
+    extensions_with_dot_to_language = {}
+    for language, extensions in extension_types.items():
+        for extension in extensions:
+            extensions_with_dot_to_language['.%s' % extension] = language
```

Comment:
@dirk-thomas nit^2: consider turning this into a dictionary comprehension.

Reply:
Not sure if the double for dict comprehension is easier to read for most people but sure: f82b7dd95bf94bc82864952b41cad426fe27b380.